components/CardDetail/CardDetail.js

```diff
@@ -105,14 +104,32 @@ export default function CardDetails() {
       </Fieldsets>
 
       <SaveButtonDiv>
-        <Button aria-label="save details" type="submit">
+        <SubmitButton aria-label="save details" type="submit">
           Save Results
-        </Button>
+        </SubmitButton>
```

Comment:
Hi Phillipp, in the vercel preview this "Save-Results" Button doesn't work. Can you check if the link wrong or if something is wrong with the button?

Reply:
Ah I just saw, that this is probably intended? 
"You can write stuff in the forminput of the DetailCard Component (no functionality yet expect console.logging the input data)"

Reply:
Hey Cathrin, yes this is "intended" :D. The save button will work with the next user story. I did not plan this well :/